### PR TITLE
television 0.5.3 (new formula)

### PR DESCRIPTION
**Pull request by p-linnane**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/t/television.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/tv -V")
+
+    pid = spawn bin/"tv", [:out, :err] => "#{testpath}/output"
+    sleep 2
```
Comment (SMillerDev) [inline Formula/t/television.rb:19]:
```suggestion
    sleep 10
```

**Comment by p-linnane** [comment]
The TUI output is not consistent in testing. Would need to drop that part of the test and just do a version check to ship this. Thoughts @chenrui333 @SMillerDev?

**Comment by chenrui333** [comment]
I tested, it seems working fine in my local, weird that it does not work in CI

**Review by chenrui333 — APPROVED** [review]

**Review by daeho-ro — APPROVED** [review]

**Comment by p-linnane** [comment]
Thank you @chenrui333!

**Pull request merged**